sphinx/source/docs/user_guide/tools.rst

```diff
@@ -581,7 +581,7 @@ If no formatter is specified for a column name, the default ``"numeral"``
 formatter is assumed.
 
 Note that format specifications are also compatible with column names that
-have spaces. For example ```@{adjusted close}{($ 0.00 a)}`` applies a format
+have spaces. For example, ```@{adjusted close}{($ 0.00 a)}`` applies a format
```

Comment:
extra backtick on the front

Reply:
This error seems to have been around for quite a while, good catch! 😀